src/library/features/crates/cratefeature.cpp

```diff
@@ -413,8 +458,21 @@ void CrateFeature::slotDeleteCrate() {
             qWarning() << "Refusing to delete locked crate" << crate;
             return;
         }
-        if (m_pTrackCollection->deleteCrate(crate.getId())) {
+        // better deletion requeried
+        if (m_pCrates->hierarchy().hasChildren(crate.getId())) {
+            if (QMessageBox::question(
+                  nullptr,
+                  tr("Deleting multiple crates"),
+                  tr("Deleting this crate will also delete all it's childer. Are you sure?"),
```

Comment:
If the term "subcrate" is acceptable then we should consistently use this naming, e.g. "Create New Subcrate".